fe/fe-core/src/main/java/org/apache/doris/planner/PlanNode.java

```diff
@@ -353,6 +327,11 @@ public List<Expr> getConjuncts() {
         return conjuncts;
     }
 
+    @Override
+    public List<? extends PlanStats> getChildrenStats() {
+        return children;
```

Comment:
why return children here? i think this interface should be used to return children's StatsDeriveResult

Reply:
It chould get StatsDeriveResult by `PlanStats::getStatsDeriveResult`.

Reply:
done